.circleci/config.yml

```
@@ -42,7 +42,11 @@ jobs:
             linkchecker -t 1 ~/build/site/
             # check external separately by pointing to all *html so no
             # failures for local file:/// -- yoh found no better way,
-            linkchecker -t 1 --check-extern --ignore-url 'file:///.*' --ignore-url https://fonts.gstatic.com ~/build/site/*html ~/build/site/*/*.html
+            linkchecker -t 1 --check-extern \
+              --ignore-url 'file:///.*' \
+              --ignore-url https://fonts.gstatic.com \
+              --ignore-url https://github.com/bids-standard/bids-specification/(pull|tree)/.* \
```

Looks like bash is trying to interpret this.

```suggestion
              --ignore-url "https://github.com/bids-standard/bids-specification/(pull|tree)/.*" \
```